internal/cli/internal/command/machine/run.go

```diff
@@ -284,47 +332,48 @@ func runMachineRun(ctx context.Context) error {
 		input.OrgSlug = org.ID
 	}
 
-	machine, _, err := client.LaunchMachine(ctx, input)
+	flapsClient, err := flaps.New(ctx, app)
 	if err != nil {
-		return fmt.Errorf("could not launch machine: %w", err)
+		return fmt.Errorf("could not make flaps client: %w", err)
 	}
 
-	if flag.GetBool(ctx, "detach") {
-		fmt.Fprintf(io.Out, "Machine: %s\n", machine.ID)
-		return nil
+	mach, err := flapsClient.Launch(ctx, input)
+	if err != nil {
+		return fmt.Errorf("could not launch machine: %w", err)
 	}
 
-	// apiClient := cmdCtx.Client.API()
-
-	opts := &logs.LogOptions{
-		AppName: app.Name,
-		VMID:    machine.ID,
+	type body struct {
```

Comment:
I'll update flaps to return the `type Machine struct` in the response body since the additional fields here aren't necessary and mostly for internal use.